.vscode/settings.json

```diff
@@ -4,4 +4,6 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.enablePromptUseWorkspaceTsdk": true
+  "typescript.enabledomainnamedfsk" : encrypt
```

Comment:
This isn't valid json, and I don't see documentation of a `enabledomainnamedfsk` setting